Ad/intern page

outstandy opened the pull request:
the ol' intern-a-rooski

👩‍🎓 Application criteria
📂 Dates + application form
📚 Learning resources

Commit:
intern page stuff

Commit:
pulling out header to handle different apply URLs, adding form link

mordecai174 commented on assets/css/_global.css:
```diff
@@ -178,6 +188,15 @@ a:not(.button) {
 }
 
+.triangle {
+  /* background-image: url('../img/triangle-huge.png'); */
```
is this supposed to be commented out?

jssee commented on assets/css/_global.css:
```diff
@@ -178,6 +188,15 @@ a:not(.button) {
 
+.triangle {
+  /* background-image: url('../img/triangle-huge.png'); */
+  /* background-size: cover; */
```
can we drop this commented code?

jssee commented on assets/css/_carousel.css:
```diff
@@ -26,6 +26,10 @@
   }
 }
 
+.no-carousel {
```
I think a better solution would be to conditionally load the carousel script on the pages it is being used. This way we wont have useless JS blocking the load time on the internship page.

mordecai174 commented on assets/css/_global.css:
```diff
@@ -74,7 +74,7 @@ a {
 }
 
-a:not(.button) {
+a:not(.button, .resource-block) {
```
I noticed there's another rule just for `.resource-block` what do you need to inherit from here?

outstandy replied:
@mordecai174 the resource blocks are all anchor links; the base link styles overwrite them.

jssee commented on views/includes/internships/_hiring.html:
```diff
@@ -0,0 +1,33 @@
+      <h3 class="beta fade-in an-beta">Who should apply?</h3>
+      <p class="sub-text fade-in an-alpha lead">Every designer starts out differently. We don’t require a specific background or experience level, but there’re a few things you should be be confident in:</p>
+      <ul class="content-list sub-text">
+        <li class="fade-in an-alpha">Your graphic and visual design skills</li>
```
I like that you're animating the `<li>` elements in individually here, but the responsibility section below doesnt do the same. Can we animate those in as well so the behavior is consistent?

mordecai174 replied:
The timing on some of the sections seems off. I know it's probably just some growing pains from trying to to apply the index page styles to this page. We need to revisit these at some point. If there's anything quick and easy to do in the meantime, let's hit it. If not we can come back to it.

jssee reviewed (changes requested):
Nice work, just have a few comments. I know we are trying to get this out soonish so feel free to disregard some of these and we can get back to it later.

There are also some design tweaks we might want to consider:
1. Theres something weird happening where the header background and the `.content` / `.hero` are slightly different shades of black
2. The intro section where the wavy triangle is under the text is hard to read, we could probably drop the opacity on the image. could just be me?
3. Theres some areas that i think we can add some type of visual hierarchy, like to the resource blocks and the work section, again this doesnt have to be addressed immediately

Commit:
cleanup and fixes

jssee committed:
[fix] Allow carousel functions to early exit if elements dont exist on
page

we can probably go back and figure out a more elegant solution for this,
and script loading in general, this works for now though and allows us
to not have to load and hide blank elements on the page.

jssee committed:
[chore] remove uneeded carousel patch
